cypress/tests/e2e/overview.cy.ts

```diff
@@ -0,0 +1,48 @@
+import { welcomeCheckbox } from '../../views/selector';
+import { tab } from '../../views/tab';
+
+describe('Close the welcome modal', () => {
+  before(() => {
+    cy.login();
+    cy.visit('');
+    cy.wait(20000);
+  });
+
+  it('close the welcome modal', () => {
+    // sometimes the welcome is presenting before move to Overview
+    cy.get('body').then(($body) => {
+      if ($body.text().includes('Do not show this again')) {
+        cy.get(welcomeCheckbox).check();
+      }
+    });
+
+    cy.visitOverview();
+    cy.wait(10000);
+    cy.get('body').then(($body) => {
+      if ($body.text().includes('Do not show this again')) {
+        cy.get(welcomeCheckbox).check();
+      }
+    });
```

Comment:
This part repeats twice, can we encapsulate this into a function?

Reply:
done